src/evm/vm.rs

```diff
@@ -454,6 +457,30 @@ where
         result
     }
 
+    /// custom spec id run_inspect
+    pub fn run_inspect(
+        &mut self,
+        mut interp: &mut Interpreter,
+        mut state: &mut S,
+    ) -> InstructionResult {
+        match self.spec_id {
```

Comment:
put SpecId::Latest at the front. Sort by usage so we don't need to do too much comparisons.